scripts/populate_panels/load_g2p_data.pl

```diff
@@ -207,8 +213,18 @@ =head1 OPTIONS
   }
 
   next if (!add_new_entry_to_panel($panel));
-
-  $entry = "$gene_symbol; $disease_name; $DDD_category; $allelic_requirement; $mutation_consequence; Target panel: $g2p_panel";
+  if (($cross_cutting_modifier) && ($mutation_consequence_flag)){
```

Comment:
It would be clearer to define the basic entry then append the optional values.

Something like 
$entry = "$gene_symbol; $disease_name; $DDD_category; $allelic_requirement; $mutation_consequence; Target panel: $g2p_panel";

$entry .=  ' Cross cutting modifier: $cross_cutting_modifier;' if $cross_cutting_modifier;

etc.